probes/src/main/java/cloud/fogbow/probes/core/services/DataProviderService.java

```diff
@@ -96,12 +96,11 @@ public DataProviderService() {
     }
 
     public Long[] getLatencies(Timestamp timestamp, boolean firstTimeAwake) {
-        List<AuditableOrderStateChange> openEvents = getOpened(timestamp, firstTimeAwake);
         List<AuditableOrderStateChange> fulfilledEvents = getFulfilled(timestamp, firstTimeAwake);
 
-        Long computeLatency = computeLatencies(getEventsOfType(openEvents, ResourceType.COMPUTE), getEventsOfType(fulfilledEvents, ResourceType.COMPUTE));
-        Long networkLatency = computeLatencies(getEventsOfType(openEvents, ResourceType.NETWORK), getEventsOfType(fulfilledEvents, ResourceType.NETWORK));
-        Long volumeLatency = computeLatencies(getEventsOfType(openEvents, ResourceType.VOLUME), getEventsOfType(fulfilledEvents, ResourceType.VOLUME));
+        Long computeLatency = computeLatencies(getEventsOfType(fulfilledEvents, ResourceType.COMPUTE));
+        Long networkLatency = computeLatencies(getEventsOfType(fulfilledEvents, ResourceType.NETWORK));
```

Comment:
Call this operation of `computeLatencies` may be misunderstood of the resource type `ResourceType.COMPUTE`. May this replace by "calculate" or a simple get.

Reply:
[Other synonyms ](https://www.google.com/search?q=computing+synonym&oq=computing+syn&aqs=chrome.1.69i57j0l5.5999j0j7&sourceid=chrome&ie=UTF-8) of `compute`

Reply:
Renamed computeLatencies to calculateLatency